tests/models/gemma2/test_modeling_gemma2.py

```diff
@@ -102,41 +102,62 @@ def setUpClass(cls):
             cls.cuda_compute_capability_major_version = torch.cuda.get_device_capability()[0]
 
     @require_read_token
-    def test_model_2b_bf16(self):
+    def test_model_9b_bf16(self):
         model_id = "google/gemma-2-9b"
         EXPECTED_TEXTS = [
-            "<bos>Hello I am doing a project for a class and I am trying to use the <code><a-image></code>",
-            "<pad><pad><bos>Hi today. So, I'm going to show you how to do a problem from the textbook. So",
+            "<bos>Hello I am doing a project on the 1918 flu pandemic and I am trying to find out how many",
+            "<pad><pad><bos>Hi today I'm going to be talking about the history of the United States. The United States of America",
         ]
 
-        model = AutoModelForCausalLM.from_pretrained(model_id, low_cpu_mem_usage=True, torch_dtype=torch.bfloat16).to(
-            torch_device
-        )
+        model = AutoModelForCausalLM.from_pretrained(
+            model_id, low_cpu_mem_usage=True, torch_dtype=torch.bfloat16, attn_implementation="eager"
```

Comment:
Note: BF16 + attn_implementation="eager" NOT set results in the model not being able to handle left-padding 🤯